### ISSUE TITLE
When facing north all entities turn invisible 

### ISSUE THREAD

**Issue by BreadMan7280** [IrisShaders/Iris#1552]
so i do use iris with some other mods but even when removing them it still has this issue, also its anything involving north so N, NE, & NW, you can also see the rough outline and the second shadow where my friend is supposed to be
![unknown](https://user-images.githubusercontent.com/100550202/180599398-cc7b9d55-9f6c-4f00-80d9-9ed5cd0f7876.png)

heres the mods that i use:

![image](https://user-images.githubusercontent.com/100550202/180599458-af329970-b72a-4ce7-90f3-6b1fb3b7c257.png)

**Comment by coderbot16** [comment]
Please remove all mods except for Sodium and Iris, then test to see if the issue persists. And please use the issue form next time!

**Comment by Jonah06443** [comment]
I'm having a similar issue and I believe what's breaking iris is the EMC-F-v2, or in simpler terms, Aristois.
![image](https://user-images.githubusercontent.com/109990807/180866796-197ee639-4229-44e7-a524-924f5ee73b48.png)

**Comment by IMS212** [comment]
We don’t support hacked clients, so that could very well be it.
